cg/cli/workflow/microsalt/base.py

```diff
@@ -275,15 +212,14 @@ def start_available(context: click.Context, dry_run: bool) -> None:
 def upload_analysis_vogue(context: click.Context, unique_id: str, dry_run: bool) -> None:
     """Upload the trending report for latest analysis of given case_id to Vogue"""
 
-    vogue_api = VogueAPI(context.obj)
-    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
-    case_obj = microsalt_analysis_api.db.family(unique_id)
+    analysis_api: MicrosaltAnalysisAPI = context.obj["analysis_api"]
```

Comment:
Should not upload stuff to vogue be under `cg/cli/vogue` ?

Reply:
Right now theres a crontab for this. So changing this would be different PR. Or this will be further complicated with servers PR as dependency